### PR TITLE
Link to repo inverted index App, set up with linters

### PR DESCRIPTION
**Pull request by desaawa**
#### What does this PR do?
This provides a link to the repository for the inverted index project, with  linters set up
#### Description of Task to be completed?
Create the repository for the proposed project and set up linters
#### How should this be manually tested?
By visiting the repository [here](https://github.com/andela-aawa/inverted-index/pull/3)
#### Any background context you want to provide?
 Linters help us to write clean code, which will be easy to read and understand
#### Screenshots (if appropriate)
eslint1
eslint2
houndcli-eslint
#### Questions:
N/A

### CODE REVIEW

**Comment by ghost** [comment]
Good basic setup

**Comment by victor-enogwe** [comment]
LGTM

**Comment by q-ode** [comment]
Kindly improve the `ReadeMe` of you repository according to the guidelines.

**Comment by desaawa** [comment]
Okay @q-ode. Noted.

**Comment by desaawa** [comment]
Hi @q-ode, I improved the `ReadMe` as per your request, and I have also set up linters for my app

**Comment by q-ode** [comment]
ReadMe is still barely descriptive.

File: 06-building-a-program-start-to-finish/setting-up-linters/output/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#### Output
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-linters/output/README.md:1]: Include links to the images in this file.

Comment (desaawa) [reply 06-building-a-program-start-to-finish/setting-up-linters/output/README.md:1]: Okay @q-ode. I have done both. Improved the ReadMe and added the links to the images.

**Pull request merged**